LoadFiles.py

```diff
@@ -1,333 +1,363 @@
+          else:
+            # Get basenames and order them, otherwise folders will distort order
+            tmpDict = { path: os.path.splitext( os.path.basename( path ) )[0] for path in self.lstFilesToLoad }
+            # We revert the order to load single layers from the back, so any
+            #  new layer will be added to the top of the layer tree
+            self.lstFilesToLoad = [k for k,v in sorted(tmpDict.items(), key=lambda item:locale.strxfrm(str(item[1])), reverse=not self.bReverseSort)]
+
+        for layerPath in self.lstFilesToLoad:
+          QApplication.processEvents()
+          if not self.progressBar.parent().parent().processStatus:
+            # The process was canceled
+            self.iface.mapCanvas().setRenderFlag( True )
+            return False
+
+          # Finally add the layer and apply the options the user chose
+          if self.bGroups:
+            group = self.tree.addGroup( os.path.dirname( layerPath ) )
+
+          baseName = os.path.basename( layerPath )
+          layerName = os.path.splitext( baseName )[ 0 ]
+          if '|layername=' in baseName and not baseName.endswith( '|layername=' ):
+            if self.bAddParentLayerName:
+              layerName = "".join( [layerName, " ", os.path.basename( layerPath ).split( '|layername=' )[1]] )
+            else:
+              layerName = baseName.split( '|layername=' )[1]
+
+          ml = self.createLayer( layerPath, layerName )
+          if ml.isValid():
+            layersLoaded += 1
+            if self.bGroups:
+              self.tree.addLayerToGroup( ml, group )
+            else:
+              self.tree.addLayer( ml, self.bLayersOff )
+            # Look whether style to load 
+            if self.bStyles:
+              styleFound="False"
+              # Has layer a style to apply ?
+              styleFile = os.path.join( baseName, os.path.splitext( baseName )[ 0 ] + ".qml")
+#              QgsApplication.messageLog().logMessage(
+#              "Has layer '{0}' a qml file {1}".format(layerPath, styleFile),
+#              "Load Them All", Qgis.Info)
+              if ( os.path.exists( styleFile)):
+                ml.loadNamedStyle( styleFile)                                
+                styleFound="True"
+              elif self.bGroups:
+                # Has the group a style to apply ?
+                aGroup = os.path.dirname( layerPath )
+                aBaseGroup = os.path.basename( aGroup )
+                styleFile = os.path.join( aGroup, aBaseGroup + ".qml")
+#                QgsApplication.messageLog().logMessage(
+#                "Has group '{0}' a qml file {1}".format(aGroup, styleFile),
+#                "Load Them All", Qgis.Info)
+                if ( os.path.exists( styleFile)):
+                  ml.loadNamedStyle( styleFile)
+                  styleFound="True"
+            if styleFound == "True":
```

Comment:
If `self.bStyles` is `False`, `styleFound` is not set before line #193. Therefore, please set `styleFound` before line #172. 
Also, use proper boolean values instead of strings for `styleFound`.